Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -92,7 +104,7 @@ open class _ChatChannelListItemView<ExtraData: ExtraDataTypes>: _ChatChannelSwip
         ]
         
         // Avatar view
-        constraintsToActivate += [
+        layout.avatarViewConstraints += [
```

Comment:
IMO we should `= []` rather than `+= []`

Reply:
good point, missed it